src/web/src/app/space/detail/component.ts

```diff
@@ -0,0 +1,277 @@
+import { Component, OnInit, ViewChild, Inject, Injectable } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { MatPaginator, PageEvent } from '@angular/material/paginator';
+import { DOCUMENT } from '@angular/common';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+
+import { Space } from '../model';
+import { SpaceInvitation } from '../model';
+import { SpaceLink } from '../model';
+import { SpaceUser } from '../model';
+
+import { SpaceService } from '../service';
+import { UserService } from '../../user/service';
+import { RequestWatcherService } from '../../request-watcher/service';
+
```

Comment:
Мы можем расширить тип который приходит с сервера, дополнительными значениями которые нам нужны в этом компоненте.

```suggestion
import { RequestWatcherService } from '../../request-watcher/service';

type SpaceUserFull = SpaceUser & {
	displayName: string,
	login: string,
	email: string,
};
```

Reply:
А в SpaceListComponent мы тоже можем расширить типы SpaceInvitation и LinkInvitation, чтобы в них было имя пространства (например, "Пространство №1")?

Reply:
Готово.